python/rmm/_cuda/gpu.py

```diff
@@ -190,12 +155,7 @@ def deviceGetName(int device):
     and status code.
     """
 
-    cdef char[256] device_name
-    cdef CUresult status = cuDeviceGetName(
-        device_name,
-        sizeof(device_name),
-        device
-    )
-    if status != 0:
+    status, device_name = cuda.cuDeviceGetName(256, cuda.CUdevice(device))
+    if status.value != 0:
```

Comment:
Could we compare to `CUDA_SUCCESS` instead?

```suggestion
    if status != cuda.CUresult.CUDA_SUCCESS:
```

I think this is the only place where we're manually checking a value instead of comparing to the enum.